releasenotes/notes/fix-exception-replay-loop-6d50b7bf1eedeaa6.yaml

```
@@ -0,0 +1,5 @@
+---
+fixes:
+  - |
+    exception replay: fixed an infinite loop that could cause memory leaks when capturing exceptions, and improved
+    overall speed and memory performance.
```

```suggestion
fixes:
  - |
    exception replay: fixed an infinite loop that could cause memory leaks when capturing exceptions
other:
  - |
    exception replay: improved overall speed and memory performance.
```


you don't have to accept this, just showing we can structure multiple/differing release notes in one file